### PR TITLE
Update .gitattributes

### PR DESCRIPTION
**Pull request by ruudk**
Screenshot 2024-11-05 at 11 40 26@2x

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -8,3 +8,7 @@ phpstan.neon export-ignore
 phpunit.xml.dist export-ignore
 psalm.xml export-ignore
 rector.php export-ignore
+box.json.dist export-ignore
+composer-dependency-analyser.php export-ignore
+scoper.inc.php export-ignore
+docs export-ignore
```
Comment (VincentLanglet) [inline .gitattributes:14]: Isnt it useful to keep docs ?
Especially because there are listed in the readme

Comment (ruudk) [reply .gitattributes:14]: I looked at Twig and they don't it. 

There might be a case where this is useful sometimes. But we should be conservative with what we archive and download every time on CI ;)

Comment (VincentLanglet) [reply .gitattributes:14]: Ok, can you update it too
```
.* export-ignore
patches export-ignore
docs export-ignore
tests export-ignore
infection.json export-ignore
Makefile export-ignore
box.json.dist export-ignore
composer-dependency-analyser.php export-ignore
phpstan.neon export-ignore
phpunit.xml.dist export-ignore
psalm.xml export-ignore
rector.php export-ignore
scoper.inc.php export-ignore
```
this way it's in the same order than in the repo, and

`composer-require-checker.json export-ignore` can be removed

Comment (ruudk) [reply .gitattributes]: Done

**Review by VincentLanglet — APPROVED** [review]

**Pull request merged**